packages/test/test-service-load/src/leaderElection.ts

```diff
@@ -38,13 +39,14 @@ export class LeaderElection {
     private runLeaderElection() {
         if (this.leaderId !== undefined && this.leaderId === this.dataStoreRuntime.clientId) {
             this.dataStoreRuntime.submitSignal("leaderMessage", "leaderMessage");
-            this.lastPinged = Date.now();
+            this.updateLastPinged();
         }else if(this.leaderId === undefined) {
-            this.logger.sendErrorEvent({eventName: "LeaderUndefinedEventError"});
+            this.logger.sendTelemetryEvent({eventName: "LeaderUndefinedEventError"});
         }else {
             const current = Date.now();
             if(this.lastPinged !== undefined && current - this.lastPinged > this.leaderWait) {
                 this.logger.sendErrorEvent({eventName: "LeaderLostEventError"});
```

Comment:
```suggestion
                this.logger.sendTelemetryEvent({eventName: "LeaderLostEventError"});
```
rather promote after we see it clean